api/Performance.json

```
@@ -757,13 +855,13 @@
               "version_added": "62"
             },
             "edge": {
-              "version_added": null
+              "version_added": "16"
             },
             "firefox": {
-              "version_added": "59"
+              "version_added": "53"
             },
             "firefox_android": {
-              "version_added": "59"
+               "version_added": "53"
```

```suggestion
              "version_added": "53"
```

Extra space got introduced here. Once this is fixed, I think this can be merged. Thanks!